miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/AbstractInstituteDefaultsController.java

```diff
@@ -113,6 +115,8 @@ public void setSecurityManager(SecurityManager securityManager) {
   }
 
   protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) throws IOException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "mapper". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)